receiver/tlscheckreceiver/factory.go

```diff
@@ -26,10 +27,13 @@ func NewFactory() receiver.Factory {
 }
 
 func newDefaultConfig() component.Config {
+	cfg := scraperhelper.NewDefaultControllerConfig()
+	cfg.CollectionInterval = 60 * time.Second
```

Comment:
This seems like unrelated change of the default behavior. Should go in a separate PR

Reply:
👍